java/flight/flight-sql/src/test/java/org/apache/arrow/flight/sql/example/FlightSqlExample.java

```diff
@@ -1488,7 +1488,14 @@ public void getStreamSchemas(final CommandGetDbSchemas command, final CallContex
   @Override
   public FlightInfo getFlightInfoTables(final CommandGetTables request, final CallContext context,
                                         final FlightDescriptor descriptor) {
-    return getFlightInfoForSchema(request, descriptor, Schemas.GET_TABLES_SCHEMA);
+
+    Schema schemaToUse = Schemas.GET_TABLES_SCHEMA;
+
+    if (!request.getIncludeSchema()) {
+      schemaToUse = Schemas.GET_TABLES_SCHEMA_NO_SCHEMA;
+    }
+
+    return getFlightInfoForSchema(request, descriptor, schemaToUse);
```

Comment:
nit: no need for so many newlines
```suggestion
    Schema schemaToUse = Schemas.GET_TABLES_SCHEMA;
    if (!request.getIncludeSchema()) {
      schemaToUse = Schemas.GET_TABLES_SCHEMA_NO_SCHEMA;
    }

    return getFlightInfoForSchema(request, descriptor, schemaToUse);
```